includes/admin/class-admin.php

```diff
@@ -515,9 +515,16 @@ public function admin_menu_combine() {
 			$cpt_menu   = 'edit.php?post_type=' . WPBDP_POST_TYPE;
 			$admin_menu = $this->menu_id;
 
-			if ( isset( $submenu[ $cpt_menu ] ) && isset( $submenu[ $admin_menu ] ) ) {
+			$current_user = wp_get_current_user();
+			$roles        = $current_user->roles;
+			$role         = ! empty( $roles ) && is_array( $roles ) ? array_shift( $roles ) : '';
```

Comment:
I think this would be a lot cleaner if we can use a capability instead of a role. Here's a list of options. I'm thinking [edit_others_pages](https://wordpress.org/support/article/roles-and-capabilities/#edit_others_pages) would work well here.

Reply:
@stephywells I revert the previous codes and just committed the new codes.